app/models/classification.rb

```diff
@@ -217,7 +224,7 @@ def self.find_assigned_entries(obj, ns = DEFAULT_NAMESPACE)
     end
 
     tag_ids = obj.tagged_with(:ns => ns).collect(&:id)
-    where(:tag_id => tag_ids) rescue []
+    where(:tag_id => tag_ids)
```

Comment:
I'm curious how the original code could ever blow up (i.e. why was the rescue introduced originally?)

Reply:
@Fryguy not sure, Maybe older versions of Rails choked on explicit nils? Also, see the comments from @himdel :)